app/jobs/wakes/update_facebook_metrics_job.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+class Wakes::UpdateFacebookMetricsJob < ActiveJob::Base
+  queue_as :wakes_metrics
```

Comment:
@benhutton 

Regarding moving the job to the app, the job's are mostly a wrapper around the API, and also seem a bit tied to the use case and the infrastructure of the app. I was thinking about creating a separate job to update facebook metrics of all our wakes locations along with one that updates only those that are passed in to it. Also decisions regarding queuing might be best taken on the app side of things because individual app rate limits could affect those decisions.